src/Components/Common/CommonComponents/EmailForm.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
+import { generalFunction } from '../../../assets/Config/generalFunction';
+import emailjs from "@emailjs/browser";
+import Button from './Button';
+
+const EmailForm = ({handleCancel}) => {
+    const [subject, setSubject] = useState('');
+    const [name, setName] = useState('');
+    const [toEmail, setToEmail] = useState('');
+    const [fromEmail, setFromEmail] = useState('');
+    const [message, setMessage] = useState('');
+    const [errors, setErrors] = useState({});
+
+    const validate = () => {
+        const errors = {};
+        if (!subject) errors.subject = "Subject is required";
+        if (!name) errors.name = "Name is required";
+        if (!fromEmail) {
+            errors.fromEmail = "From email is required";
+        } else if (!/\S+@\S+\.\S+/.test(fromEmail)) {
+            errors.fromEmail = "From email is invalid";
+        }
+        if (!toEmail) {
+            errors.toEmail = "To email is required";
+        } else if (!/\S+@\S+\.\S+/.test(toEmail)) {
+            errors.toEmail = "To email is invalid";
+        }
+        if (!message) errors.message = "Message is required";
+        return errors;
+    };
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+
+        const validationErrors = validate();
+        if (Object.keys(validationErrors).length > 0) {
+            setErrors(validationErrors);
+            return;
+        }
+
+        const serviceId = 'service_f1v49vl';
+        const templateId = 'template_kbi801d';
+        const publicKey = 'BJqAtQHmSB8T5inCK';
+        const privateKey = '2UNvN5729M3b_VcF6Xwqj';
```

Comment:
what keys are these? Thinking if we should keep it here or move to './Config/appConfig'

Reply:
For now we could keep them here but lets add a comment where to get them from in future